src/components/inputs/money-input/money-input.js

```diff
@@ -328,7 +349,9 @@ class MoneyInput extends React.Component {
     onFocus: PropTypes.func,
     onBlur: PropTypes.func,
     isDisabled: PropTypes.bool,
-    onChange: PropTypes.func.isRequired,
+    isReadOnly: PropTypes.bool,
+    isAutofocussed: PropTypes.bool,
+    onChange: requiredIf(PropTypes.func, props => !props.isReadOnly),
```

Comment:
Curious: is this a general pattern across other inputs or what we strive for and should change elsewhere to if not already?

Reply:
it's the same in `LocalizedTextInput`, and quite a few inputs have quite a bit of copy / paste from that input